store/shared/migrate/mysql/ddl_gen.go

```diff
@@ -658,3 +662,18 @@ CREATE TABLE IF NOT EXISTS latest (
 var createIndexLatestRepo = `
 CREATE INDEX ix_latest_repo ON latest (latest_repo_id);
 `
+
+//
+// 015_create_table_template.sql
+//
+
+var createTableTemplate = `
+CREATE TABLE IF NOT EXISTS template (
```

Comment:
It seams the table names are names in plural.

```suggestion
CREATE TABLE IF NOT EXISTS templates (
```